playbooks/splunk_configure_dmc.yml

```diff
@@ -0,0 +1,24 @@
+---
```

Comment:
You don't really need this in the `playbooks`, you can just run `ansible-playbook -v playbooks/splunk_install_or_upgrade.yml <your inventory> -e deployment_task=configure_dmc.yml`.
Maybe we should have a `ad_hoc` playbook for these cases.

Reply:
Definitely the ad_hoc playbook would be good i think. I'll remove the playbook.

Reply:
Good point, Maybe the DMC task should include the `roles/splunk/tasks/configure_idxc_sh.yml` task

Reply:
Yeah we can fold it in there, I have no problem with that.